configuration/data/definitions.ts

```diff
@@ -23,14 +23,21 @@ export interface BaseAgentConfig {
   interval: number | string;
 }
 
+export type ProcessorConfig = BaseAgentConfig & {
+  allowed?: string[];
+  denied?: string[];
+  subsidizedRemotes?: string[];
+  s3?: S3Config;
+};
+
 export interface AgentConfig {
   rpcStyle: string;
-  timelag: number | string;
   db: string;
+  metrics: number;
   logging: LogConfig;
   updater: BaseAgentConfig;
   relayer: BaseAgentConfig;
-  processor: BaseAgentConfig;
+  processor: ProcessorConfig;
   watcher: BaseAgentConfig;
   kathy: BaseAgentConfig;
```

Comment:
Kathy config needs updating I believe. `ChatGenConfig` field is added beyond base